src/plugins/plugin.title.js

```diff
@@ -111,13 +112,15 @@ module.exports = function(Chart) {
 				globalDefaults = Chart.defaults.global,
 				display = opts.display,
 				fontSize = valueOrDefault(opts.fontSize, globalDefaults.defaultFontSize),
-				minSize = me.minSize;
+				minSize = me.minSize,
+				lineCount = helpers.isArray(opts.text) ? opts.text.length : 1,
+				textSize = display ? (lineCount * fontSize) + (opts.padding * 2) + (opts.lineSpacing * (lineCount - 1)) : 0;
```

Comment:
By default, lineHeight should be equal to fontSize, then textSize is lineHeight*lineCount. If lineHeight is 0, all lines draw on the same line.